lld/test/ELF/aarch64-reloc-gotpcrel32.s

```diff
@@ -0,0 +1,35 @@
+// REQUIRES: aarch64
+// RUN: llvm-mc -filetype=obj -triple=aarch64 %s -o %t.o
+// RUN: ld.lld %t.o -o %t.so -shared
```

Comment:
See `aarch64-reloc-plt32.s`. We should `checkInt` for R_AARCH64_GOTPCREL32

Reply:
Done.